[IMP] l10n_es: several updates incl libros de IVA, TicketBAI & FACe (#18135)
odoo/documentation: content/applications/finance/fiscal_localizations/spain.rst

   * - :guilabel:`Spain - Facturae EDI - Invoice Period`
     - `l10n_es_edi_facturae_invoice_period`
     - Patch module for Invoice Period in EDI Facturae.
   * - :guilabel:`Spain - SII EDI Suministro de Libros`
     - `l10n_es_edi_sii`
     - Sends taxes information to SII (Suministro Inmediato de Información).
   * - :guilabel:`Spain - TicketBAI`
     - `l10n_es_edi_tbai`
     - Sends invoices and vendor bills to the Diputaciones Forales of Araba/Álava, Bizkaia and
       Gipuzkoa.
   * - :guilabel:`Spain - Veri*Factu`
     - `l10n_es_edi_verifactu`
     - Sends Veri*Factu XML to AEAT (Agencia Estatal de Administración Tributaria).
   * - :guilabel:`Spain - Veri*Factu for Point of Sale`
     - `l10n_es_edi_verifactu_pos`
     - Adds Veri*Factu support to Point of Sale.
   * - :guilabel:`Spain - Modelo 130 Tax report`
     - `l10n_es_modelo130`
     - Allows Modelo 130 Tax report specifications.
   * - :guilabel:`Spain - Point of Sale`
     - `l10n_es_pos`
     - Spanish localization for Point of Sale.
   * - :guilabel:`Spain - POS + TicketBAI`
     - `l10n_es_pos_tbai`
     - Sends invoices to the Diputaciones Forales of Araba/Álava, Bizkaia and Gipuzkoa.

>    * - :guilabel:`Spain - Modelo 130`

Reviewer: I would keep related modules together, like both Modelo 130 modules. I assume listing modules alphabetically would do that in most situations :)

     - `l10n_es_reports_modelo130`
     - Allows Modelo 130 Tax report specifications.

.. note::
   The localization's core modules are installed automatically with the localization. The rest can
   be manually :doc:`installed <../../general/apps_modules>`.

.. _localizations/spain/loc-review:

Localization overview
=====================

The Spanish localization package ensures compliance with Spanish fiscal and accounting regulations.
It includes tools for managing taxes, fiscal positions, reporting, and a predefined chart of
accounts tailored to Spain’s standards.

The Spanish localization package provides the following key features to ensure compliance with
local fiscal and accounting regulations:

- :doc:`Chart of accounts <../accounting/get_started/chart_of_accounts>`: a predefined structure
  tailored to Spanish accounting standards.
- :doc:`Taxes <../accounting/taxes>` pre-configured tax rates, including standard VAT, zero-rated,
  and exempt options.

> - :doc:`Fiscal Positions <../accounting/taxes/fiscal_positions>`: automated tax adjustments based

Reviewer: ```suggestion
- :doc:`Fiscal positions <../accounting/taxes/fiscal_positions>`: automated tax adjustments based
```

     General de Contabilidad) :doc:`chart of accounts
     <../accounting/get_started/chart_of_accounts>`:

     - :guilabel:`Spain - SMEs (2008)`
     - :guilabel:`Spain - Non-profit entities (2008)`
     - :guilabel:`Spain - Cooperatives - Complete (2008)`
     - :guilabel:`Spain - Cooperatives - SMEs (2008)`
     - :guilabel:`Spain - Complete (2008)`

   - By default, new Odoo Online databases are set up with the :guilabel:`Spain - SMEs (2008)`
     fiscal localization package.

.. _localizations/spain/reports:

Reports
-------

The Spanish-specific statement reports available are:

- Balance Sheet
- Profit & Loss (ES)
- EC Sales List
- :ref:`Tax report (Modelo) <localizations/spain/tax-report>`

.. _localizations/spain/tax-report:

Tax report
~~~~~~~~~~

To access Spain-specific tax reports (Modelo), go to :menuselection:`Accounting --> Reporting -->
Tax Report`, click :icon:`fa-book` :guilabel:`Report:` and select one of the following:

- Tax Report (Mod 111)(ES)
- Tax Report (Mod 115)(ES)
- :ref:`Tax Report (Mod 130)(ES) <localizations/spain/modelo-130>`
- Tax Report (Mod 303)(ES)
- Tax Report (Mod 347)(ES)
- Tax Report (Mod 349)(ES)
- Tax Report (Mod 390)(ES)

.. tip::

>    To export the :guilabel:`VAT records books (XLSX)` (Libros de IVA), click :icon:`fa-book`

Reviewer: ```suggestion
   To export the VAT records books (Libros de IVA) in XLSX format, click :icon:`fa-book`
```
No guilabel at first since you refer to the concept :)

   :guilabel:`Report:` and select :guilabel:`Generic Tax report`. Then, click the

>    :icon:`fa-caret-down` :guilabel:`(down arrow)` and select :guilabel:`VAT records books (XLSX)` to

Reviewer: ```suggestion
   :icon:`fa-caret-down` :guilabel:`(down arrow)` and select :guilabel:`VAT Records Books (XLSX)` to
```
Here we can use the guilabel since we refer to the specific UI element, and we respect the UI's capitalization as well

     - Sends taxes information to SII (Suministro Inmediato de Información).
   * - :guilabel:`Spain - TicketBAI`
     - `l10n_es_edi_tbai`
     - Sends invoices and vendor bills to the Diputaciones Forales of Araba/Álava, Bizkaia and
       Gipuzkoa.
   * - :guilabel:`Spain - Veri*Factu`
     - `l10n_es_edi_verifactu`
     - Sends Veri*Factu XML to AEAT (Agencia Estatal de Administración Tributaria).
   * - :guilabel:`Spain - Veri*Factu for Point of Sale`
     - `l10n_es_edi_verifactu_pos`
     - Adds Veri*Factu support to Point of Sale.

.. note::
   In some cases, such as when upgrading to a version with additional modules, it is possible that
   modules may not be installed automatically. Any missing modules can be manually :doc:`installed
   <../../general/apps_modules>`.

.. _localizations/spain/loc-review:

Localization overview
=====================

The Spanish localization package ensures compliance with Spanish fiscal and accounting regulations.
It includes tools for managing taxes, fiscal positions, reporting, and a predefined chart of
accounts tailored to Spain’s standards.

The Spanish localization package provides the following key features to ensure compliance with

> local fiscal and accounting regulations:

Reviewer: I didn't comment on that for the first review since it seemed to be part of the template, and I didn't want to make you change it. However, since you are modifying the template, I would like to mention that having both:

> The Spanish localization package provides the following key features to ensure compliance with
> local fiscal and accounting regulations:

right after

> The Spanish localization package ensures compliance with Spanish fiscal and accounting regulations.

is quite redundant.
Unless I'm missing something, I would try to merge both paragraphs into one.

- :doc:`Chart of accounts <../accounting/get_started/chart_of_accounts>`: a predefined structure
  tailored to Spanish accounting standards.
- :doc:`Taxes <../accounting/taxes>` pre-configured tax rates, including standard VAT, zero-rated,
  and exempt options.
- :doc:`Fiscal positions <../accounting/taxes/fiscal_positions>`: automated tax adjustments based
  on customer or supplier registration status.
- :ref:`Reporting <localizations/spain/reports>`

.. note::
   - There are several distinct :ref:`Spanish fiscal localization packages
     <fiscal_localizations/packages>` available, each featuring its own pre-configured PGCE (Plan
     General de Contabilidad) :doc:`chart of accounts
     <../accounting/get_started/chart_of_accounts>`:

     - :guilabel:`Spain - SMEs (2008)`
     - :guilabel:`Spain - Non-profit entities (2008)`
     - :guilabel:`Spain - Cooperatives - Complete (2008)`
     - :guilabel:`Spain - Cooperatives - SMEs (2008)`
     - :guilabel:`Spain - Complete (2008)`

>    - By default, new Odoo Online databases are set up with the :guilabel:`Spain - SMEs (2008)`

Reviewer: ```suggestion
   - By default, new Odoo Online databases with Spain selected as their country are set up with the :guilabel:`Spain - SMEs (2008)`
```
I hesitated to comment, as it's ok within the context, but the sentence is technically false without that bit of information. However, my suggestion makes the sentence a bit clumsy.

   Producers of Veri*Factu billing systems must self-certify their compliance with the regulations.
   :download:`Download Odoo's "declaración responsable"<spain/declaracion_responsable.pdf>`

> **Veri*Factu** is an Verifiable Invoice Issuance System used by the Spanish Tax Agency. It is

Reviewer: ```suggestion
**Veri*Factu** is the Spanish Tax Agency's verifiable invoice issuance system. It is
```

   order. Scan this code to verify that the invoice has been received and recognized by the
   :abbr:`AEAT (Agencia Estatal de Administración Tributaria)`

.. _localizations/spain/ticketbai:

TicketBAI
=========

`Ticket BAI <https://www.gipuzkoa.eus/es/web/ogasuna/ticketbai>`_ or **TBAI** is an e-Invoicing
system used by the Basque government and its three provincial councils (Álava, Biscay, and
Gipuzkoa). Odoo supports the **TicketBAI (TBAI)** electronic invoicing format for all three regions
of the **Basque Country**.

.. note::
   To use TicketBAI, make sure:

   - :guilabel:`Country` and :guilabel:`Tax ID` fields on the :doc:`company record

>      </applications/general/companies>` are filled in.

Reviewer: ```suggestion
     <../../general/companies>` are filled in.
```
We try to use relative links if possible :)
